run.py

```diff
@@ -0,0 +1,9 @@
+from src.file_system import FileSystem 
+
+fs = FileSystem('myFile.csv')
+fs.create_file()
```

Comment:
What happens if we try to create a file with a name that already exists? 

![image](https://github.com/DeKaylaPhillips/File-System-Project/assets/17726736/86919686-f2fb-450b-8693-4ca773c46baa)

I suggest using the `w` instead of `x` 
![image](https://github.com/DeKaylaPhillips/File-System-Project/assets/17726736/fbc55ec8-192c-4398-b155-707a534810a8)

Reply:
I put this comment in the wrong place 😅 . The issue is` src/file_system.py` line 7

Reply:
Of course, like everything in code there are multiple solutions. You can also try catching the error if the file already exists. 

https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
Ahh, okay! Thank you! I've implemented the changes to the mode and refactored the test. :) Please lmk what you think! I also figured using "with open(..)" instead of `open()` and `close()`, and printing the file name for confirmation that the file was created would also be a decent direction to go in.

Reply:
Realizing I saw the updated comments super late LOL (needed to refresh). Should I use my current implementation which allows for a file to be wiped if it does exist, or do you think constraints would be in place that would make it so an admin cannot override a pre-existing file? The latter was my initial thought process (and then I'd just handle the error), but now I'm not sure which direction would be best. Right now, my implementation looks like this:

<img width="423" alt="image" src="https://github.com/DeKaylaPhillips/File-System-Project/assets/91583190/26edb53a-d23e-4f7f-aa7d-c1b621b51b70">